Data/TST_Data_Convention.md

```diff
@@ -1,51 +1,125 @@
 └───Experiment folder
 	   │   Metadata file .xls
 	   │   Test data 001 .csv
 	   │   Test data 002 .csv
 	   │   ...
-
 ```
 
+## Directory/file naming conventions
 
-### Directory/file naming conventions
-
-* Experiment folder:  `TST_{Researcher's lastname}_{Date}_{Test type}`
-* Metadata file: `TST_{Date}_{Test type}_metadata.xls`
-* Test data: `TST_{Date}_{Test type}_{###}.csv`
+- Experiment folder: `TST_{Researcher's lastname}_{Date}_{Test type}`
+- Metadata file: `TST_{Date}_{Test type}_metadata.xls`
+- Test data: `TST_{Date}_{Test type}_{###}.csv`
 
 with:
 
-* date: `YYYY-MM` (starting month of the experiment)
-* Test type: `FA` = standard fatigue or `QS` = standard quasi-static.
-* `###`: Specimen number
-
-
-
-### TST CSV files standards (column names must be exact) :
-
-
-| Column name          | Description                                        | Unit  | Data type | Mandatory  |
-|----------------------|----------------------------------------------------|-------|-----------|---|
-| Machine\_N_cycles     | Number of cycles counted by the machine            | [-]   | int       | y  |
-| Machine_Load         | Load measured by the machine                       | [kN] | double    |  y |
-| Machine_Displacement | Displacement measured by the machine               | [mm]   | double    |  y |
-| index            | Image number                                       | [-]   | int       |   |
-| Camera\_N_cycles         | Number of cycles counted by the camera             | [-]   | int       |   |
-| exx              | Strain measured along main axis              | [-]  | double    |   |
-| eyy              | Strain measured along secondary axis         | [-]  | double    |   |
-| exy              | Strain measured along a specified axis       | [-]  | double    |   |
-| crack_length     | Crack length measurement (for fracture testings)   | [mm]  | double    |   |
-| Th_time              | Time as counted by temperature monitoring          | [sec] | int       |   |
-| Th\_N_cycles          | Number of cycles counted by temperature monitoring | [-]   | int       |   |
-| Th\_specimen_max      | Maximum temperature monitored                      | [°C]  | single    |   |
-| Th\_specimen_mean     | Mean temperature                                   | [°C]  | single    |   |
-| Th_chamber           | Temperature of the test environment                | [°C]  | single    |   |
-| Th_uppergrips        | Temperature of the upper grips                     | [°C]  | single    |   |
-| Th_lowergrips        | Temperature of the lower grips                     | [°C]  | single    |   |
-| Specimen Name        | Specimen Name                    | [-]  | string    |   |
+- date: `YYYY-MM` (starting month of the experiment)
+- Test type:
+  - `FA` = standard fatigue
+  - `QS` = standard quasi-static
+  - `TM` = DMA or temperature
+- `###`: Specimen number
+
+## TST XLS file standards
+
+Must match template : [TST_Data_Template.xls](TST_Data_Template.xls)
```

Comment:
Proposed addition : 
"Note that only row 4 (sheet 1) and row 3 (sheet 2) must be filled with metadata. The rest must remain unchanged"

(This is to avoid researchers messing up with the template)

Reply:
Also I can't comment on the Data_template because it's XLS, but I suggest highlighting these rows - to avoid researchers messing up with template. 
[TST_Data_Template (3).xls](https://github.com/EPFL-ENAC/CCFatiguePlatform/files/9381959/TST_Data_Template.3.xls)

Reply:
I found how to make suggestions direclty in PRs comments - thanks to @guilbep  - see below !